pkg/bgpv1/manager/reconciler/types.go

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: this file name can be reconciler.go

Reply:
@rastislavs  feel free to remove dont-merge/discussion tag if you want to keep the change as is. It's a nit comment.

Reply:
ack, will rename